### ISSUE TITLE
bashbrew: use with local repository

### ISSUE THREAD

**Issue by fbouliane** [docker-library/official-images#4790]
I know it's not one of the intended features, but i'm looking to reuse bashbrew for personnal repositories. Currently, it's only possible with bashbrew to reference git repository as reference for an image.

Having a verb in the rfc2822 config file for targeting a local directory could be useful in my case as I would like to have the  library-file stored in the individual project rather than in a central repository.

Something like "LocalDirectory"

First, does this feature exists ? Maybe I did not look correctly.
Second, if not, would it fit the spirit of bashbrew to add such feature ? 
Is bashbrew really tied to a git repository for the tagging process ?

**Comment by tianon** [comment]
You're not wrong -- such a feature definitely doesn't exist today.  Invoking `git` is pretty centric to the way `bashbrew` functions, and using an explicit Git commit is most of the reason it exists (because otherwise, simply using `docker build` directly would be sufficient -- it can even take a Git URL directly).

**Comment by andrewnicols** [comment]
Likewise, it would be great to be able to use bashbrew to manage images in our own repositories.

I'd love to see this take an 'Organization' option, or similar.

**Comment by tianon** [comment]
@andrewnicols it sounds like what you want and what the OP is looking for are different.  You should probably look at the `--namespace` flag which is supported on all of `bashbrew build`, `bashbrew tag`, and `bashbrew push` (although `bashbrew tag` is less useful if you use `--namespace` on `bashbrew build` since the purpose of `bashbrew tag` is to take `xxx` and re-tag it as `namespace/xxx` specifically for use with `bashbrew push --namespace`).  I do exactly that for most of the actively-maintained images under https://hub.docker.com/u/tianon/.

**Comment by andrewnicols** [comment]
Thanks @tianon - that does sound like what I want.

**Comment by fbouliane** [comment]
Thanks @tianon, for now I will try to make it work on a fork.
My need is to not hardcode the path of the different dockerfiles and version a project publishes in a build script.
Would you be open for adding LocalRepository to the builder ?
